CHANGES.md

```diff
@@ -7,6 +7,8 @@ Change Log
 * Added ability to support touch event in Imagery Layers Split demo application. [#5948](https://github.com/AnalyticalGraphicsInc/cesium/pull/5948)
```

Comment:
This should be simple enough to add to czml too. The only changes would be in `CzmlDataSource`.